src/locale/locales/ca/messages.po

```diff
@@ -864,7 +864,7 @@ msgstr "Apel·la \"{0}\" etiqueta"
 #: src/view/com/util/forms/PostDropdownBtn.tsx:337
 #: src/view/com/util/forms/PostDropdownBtn.tsx:346
 #~ msgid "Appeal content warning"
-#~ msgstr "Advertència d'apel·lació sobre el contingut"
+#~ msgstr "Advertència d'apel·lació s el contingut"
```

Comment:
```suggestion
#~ msgstr "Advertència d'apel·lació sobre el contingut"
```

Reply:
Hi, just fyi the messages that start with `#~` are obsolete as they've been removed from the app, so there's no need to bother with updating them :)